embedded-aerospike/src/main/java/com/testcontainers/aerospike/AerospikeProperties.java

```diff
@@ -21,9 +21,9 @@
 * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 * SOFTWARE.
  */
-package com.playtika.test.aerospike;
+package com.testcontainers.aerospike;
```

Comment:
if you want to rename packages i'd recommend
 ```com.playtika.test.aerospike``` -> ```com.testcontainers.spring.boot.aerospike```
you may create collisions in case if you would you root ```com.testcontainers.``` package

Reply:
this is not collision

Reply:
why? i agree with @tdanylchuk

Reply:
because ```com.testcontainers.aerospike``` could be related to plain ```aerospike``` container w\o spring boot at all... same for kafka... etc...